php1/mysql/class-mail.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+session_start();
+
+use GuzzleHttp\Client;
+
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\SMTP;
+use PHPMailer\PHPMailer\Exception;
+
+require '../../vendor/autoload.php';
+
+/**
+ * Validating the email Id then verifying the email
+ * 
+ */ 
+class Mail {
```

Comment:
I think in case of class we write '{' from the next line! So please share your opinion.

Reply:
I have cross-checked your opinion and it is correct, I will change it.